Selenium/EmployeeRegister.py

```diff
@@ -0,0 +1,132 @@
+from Classes.EmployeeRegistrationData import EmployeeRegistrationData
+from utils.utils import *
+from Selenium.UserLogin import test_user_login
+
+ip = get_variable_value("IP_FRONTEND")
+delay = int(get_variable_value("DELAY"))
+home_path = get_variable_value("HOME_PATH")
+home_url = f"http://{ip}{home_path}"
+
+# employee_management_path = get_variable_value("EMPLOYEE_MANAGEMENT")
+# employee_management_url = f"http://{ip}{employee_management_path}"
+restaurants_management_path = get_variable_value("RESTAURANTS_MANAGEMENT")
+restaurants_management_url = f"http://{ip}{restaurants_management_path}"
+sample_data = EmployeeRegistrationData.generate_employee_data()
+
+def test_register_employee(driver):
+    test_user_login(driver, False)
+
+    info("EMPLOYEE REGISTER TEST")
+    try:
+        # Sprawdzenie tytułu
+        check_page_title(driver, "React App")
+
+        # Sprawdzenie czy istnieje elemenet o id root, w naszym przypadku div
+        wait_for_element(driver, By.ID, "root")
+
+        #EMPLOYEE REGISTER
+        # Znalezienie i kliknięcie przycisku
+        click_button(driver, By.ID, "NavbarRestaurantsSectionButton")
+
+        # Czekamy na zmianę strony
+        wait_for_url_to_be(driver, restaurants_management_url)
+        click_button(driver, By.ID, "menu-listItem-employees-button")
+
+        # czekamy na załadowanie tabelki
+        # to do
```

Comment:
Dodajmy tutaj tylko adnotację #TODO żeby później nam to nie uciekło "#TODO" na pewno jest rozpoznawany przez pycharm jeśli "# to do" też to nie trzeba zmieniać